WordPress/Classes/ViewRelated/Post/PostEditor+Publish.swift

```diff
@@ -374,13 +373,13 @@ extension PostEditor where Self: UIViewController {
         }
     }
 
-    func dismissOrPopView(didSave: Bool, shouldShowPostEpilogue: Bool = true) {
+    func dismissOrPopView() {
         stopEditing()
 
         WPAppAnalytics.track(.editorClosed, withProperties: [WPAppAnalyticsKeyEditorSource: analyticsEditorSource], with: post)
 
         if let onClose = onClose {
-            onClose(didSave, shouldShowPostEpilogue)
+            onClose(true, false)
```

Comment:
I'm resisting the urge to suggest further refactors, since this is so hard to follow, but if I had to pick just one, this method could really benefit from some argument labels.

Reply:
Yes, I felt the same way while writing the code, there are a ton of improvements that could be made here, but on this PR I'm just trying to fix the bug.